### PR TITLE
reCAPTCHA V2 implemented

### PR DESCRIPTION
**Pull request by Vojtech-Sassmann**
* Implemented new version of reCAPTCHA. These changes also require
changes in PERUN RPC.

### CODE REVIEW

**Comment by Vojtech-Sassmann** [comment]
The pull request with server-side changes: https://github.com/CESNET/perun/pull/1761

**Comment by Vojtech-Sassmann** [comment]
I have made the changes you requested.

File: pom.xml [hunk 1]
```diff
@@ -36,7 +36,7 @@
 		<gwtPluginVersion>2.8.1</gwtPluginVersion>
 		<gwtp.version>1.6</gwtp.version>
 		<gin.version>2.1.2</gin.version>
-
+		<agnie.version>1.0-SNAPSHOT</agnie.version>
```
Comment (zlamalp) [inline pom.xml]: Please remove version for reCaptcha, since it's now part of the code.

File: pom.xml [hunk 2]
```diff
@@ -287,6 +287,11 @@
 				<enabled>false</enabled>
 			</releases>
 		</repository>
+		<!--Captcha wrapper repository-->
```
Comment (zlamalp) [inline pom.xml]: Please remove mvn repo for reCaptcha, since it's now part of the code.

Comment (Vojtech-Sassmann) [reply pom.xml]: Repo removed.

File: pom.xml (hunk 1 above)
Comment (Vojtech-Sassmann) [reply pom.xml]: Removed.

File: perun-wui-registrar/src/main/java/cz/metacentrum/perun/wui/registrar/client/PerunRegistrarPresenter.java [hunk 3]
```diff
@@ -137,103 +126,67 @@ public void onLoadingStart() {
...
+								alert.setText(translation.captchaFailed());
+								alert.setVisible(true);
+							}
+						}
+
+						@Override
+						public void onError(PerunException error) {
+							loader.setVisible(false);
+							captcha.reset();
+							alert.setText(translation.serverDown());
```
Comment (zlamalp) [inline perun-wui-registrar/src/main/java/cz/metacentrum/perun/wui/registrar/client/PerunRegistrarPresenter.java]: Can you please use our ``AlertErrorReporter`` which can interpret PerunException returned by RPC and allows user to report bug from GUI ? Thank you.

Comment (Vojtech-Sassmann) [reply perun-wui-registrar/src/main/java/cz/metacentrum/perun/wui/registrar/client/PerunRegistrarPresenter.java]: Done.

**Pull request merged**